composer.json

```diff
@@ -11,37 +11,37 @@
     },
     "require" : {
         "php" : ">=7.1.0",
-        "adldap2/adldap2" : "9.1.3",
-        "adodb/adodb-php" : "5.20.12",
+        "adldap2/adldap2" : "10.0.8",
+        "adodb/adodb-php" : "5.20.14",
         "doctrine/common" : "2.10.0",
         "doctrine/couchdb" : "1.0-beta4",
         "doctrine/orm" : "2.6.3",
-        "dompdf/dompdf" : "0.8.2",
+        "dompdf/dompdf" : "0.8.3",
         "ezyang/htmlpurifier" : "4.10.0",
-        "knplabs/knp-snappy" : "1.0.4",
+        "knplabs/knp-snappy" : "1.1.0",
         "mobiledetect/mobiledetectlib": "2.8.33",
-        "mpdf/mpdf" : "7.1.7",
-        "phpmailer/phpmailer" : "6.0.6",
-        "phpoffice/phpspreadsheet": "1.5.2",
-        "phpseclib/phpseclib" : "2.0.12",
-        "rospdf/pdf-php" : "0.12.48",
+        "mpdf/mpdf" : "8.0.0",
+        "phpmailer/phpmailer" : "6.0.7",
+        "phpoffice/phpspreadsheet": "1.6.0",
+        "phpseclib/phpseclib" : "2.0.15",
+        "rospdf/pdf-php" : "0.12.49",
         "smarty/smarty" : "2.6.31",
-        "symfony/config" : "3.4.19",
-        "symfony/dependency-injection" : "3.4.19",
-        "symfony/event-dispatcher" : "3.4.19",
-        "symfony/http-foundation" : "3.4.19",
-        "symfony/yaml" : "3.4.19",
-        "twig/twig" : "2.5.0",
-        "vlucas/phpdotenv" : "2.5.1",
+        "symfony/config" : "3.4.24",
+        "symfony/dependency-injection" : "3.4.24",
+        "symfony/event-dispatcher" : "3.4.24",
+        "symfony/http-foundation" : "3.4.24",
+        "symfony/yaml" : "3.4.24",
+        "twig/twig" : "2.7.4",
+        "vlucas/phpdotenv" : "3.3.3",
         "yubico/u2flib-server" : "1.0.2",
         "zendframework/zendframework" : "2.5.3",
-        "particle/validator": "2.3.3",
+        "particle/validator": "2.3.4",
         "guzzlehttp/guzzle": "6.3.3",
-        "kamermans/guzzle-oauth2-subscriber": "1.0.5",
-        "league/omnipay": "3.0.1",
-        "academe/omnipay-authorizenetapi": "3.0.4",
+        "kamermans/guzzle-oauth2-subscriber": "1.0.6",
+        "league/omnipay": "3.0.2",
+        "academe/omnipay-authorizenetapi": "3.1.1",
```

Comment:
@sjpadgett ,
Just wanted to make you aware of these package updates. Don't appear to be any big changes (mpdf dif have some breaking changes, which I fixed).